README.md

```diff
@@ -96,19 +96,19 @@ Example:
 ![example](Images/disconnect.PNG)
 
 ### Mute Input
-Mutes audio input. Has no effect unless connected.
+Mutes audio input. The method has no effect unless an audio input device is connected.
```

Comment:
This (and several statements below) is incorrect, but also means that the initial comment was unclear in the first place. These methods have no effect unless connected to a conference. It's a little unusual but the limimitation is, AFAIK, enforced by the C++ SDK.